### ISSUE TITLE
[Keyboard Navigation - Samples- Food Order With Validation]: 'Steak', 'Chicken' & 'Tofu' Menu items present under Adaptive Cards section is not accessible with using Right\Left arrow key.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7235]
https://microsoft.visualstudio.com/OS/_workitems/edit/30943457

**Comment by vagpt** [comment]
This issue is not repro on the environment, hence closing this issue.

Test Environment:
URL: https://adaptivecards.io/
OS Version: 23H2(OS Build 25931.1000)
Browser Version: Edge Dev (118.0.2060.1)
Screen Reader: Narrator
